gateway/src/main/resources/application.yml

```
@@ -59,3 +65,11 @@ eureka:
 springboot:
   jwt:
     secret: qwertyuiopasdfghjklzxcvbnm!qwertyuiopasdfghjklzxcvbnm
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)